service/history/timerQueueProcessor2_test.go

```diff
@@ -116,13 +117,10 @@ func (s *timerQueueProcessor2Suite) SetupTest() {
 	s.mockShardManager = &mocks.ShardManager{}
 	s.mockHistoryV2Mgr = &mocks.HistoryV2Manager{}
 	s.mockVisibilityMgr = &mocks.VisibilityManager{}
-	s.mockDomainCache = &cache.DomainCacheMock{}
+
```

Comment:
remove

Reply:
i will make change to timer builder in the next PR, the test will be rewritten